Misc/NEWS.d/next/Library/2017-06-29-22-04-44.bpo-30807.sLtjY-.rst

```diff
@@ -0,0 +1,6 @@
+signal.setitimer() may disable the timer when passed a tiny value.
+
+Tiny values (such as 1e-6) are valid non-zero values for setitimer(), which
```

Comment:
Hum, I'm not sure about your Git-commit-message-like formatting. I may be needed to agree on a style on python-dev.